server/githubsrm/apis/utils.py

```diff
@@ -44,9 +44,11 @@ def verify_github_details(verify_user=False, **kwargs):
 
     if verify_user:
         response = session.get(f"{github_api}/users/{kwargs['user_id']}")
-        if response.status_code == 403:
-            response = session.get(f"{source_route}/{kwargs['user_id']}")
-        return response.status_code == 200
+        res = response.json()
+        if(res['type']=="User" and response.status_code == 200):
+            return response.status_code == 200
+        elif(res['type']!="User" and response.status_code == 200):
+            return False
```

Comment:
Create a nested `elif` instead of using the same check at two levels

Reply:
@Codered9 any updates?

Reply:
I have made the required changes. The last commit is a wip commit as local env was creating a build issue.